### PR TITLE
Cloud Updates

### PR DESCRIPTION
**Pull request by erondpowell**
Closes DOCS-170

### CODE REVIEW

**Comment by erondpowell** [comment]
@AlexGaillard It appears you owned all these issues so I tagged you in a review  to glance over the facts.

- removed "general purpose" tier verbiage and replaces with standard tier.
- removed "Lite tier" references entirely.
- updated community cloud policy to say 14 days up, then seven days for each resume.
- replaced relevant media.

File: cloud/projects.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ Standard Projects come with the following configuration options:
 - **Project URL** — Custom URL: `your-custom-url.directus.app`. _This cannot be changed_.
 - [Datacenter Region](/cloud/glossary#datacenter-regions) — `United States, East`, `Europe, Frankfurt` or
   `Asia Pacific, Singapore`.
-- [Node Type](/cloud/glossary#node-types) — Lite or General Purpose Tier Nodes.
-- [Infrastructure](/cloud/glossary#nodes) — 1-6 Reserved Nodes.
+- [Node Type](/cloud/glossary#node-types) — Standard Tier Nodes.
```
Comment (AlexGaillard) [inline cloud/projects.md:91]: As I understand it this section outlines the options you can choose from?

The Node Type selection is coming out completely so this configuration. option no longer applies.

**Review by AlexGaillard — COMMENTED** [review]
I also noticed under Manage a Standard Project section the language mentions being able to update Node Type, which is no longer the case.

File: cloud/projects.md (hunk 1 above)
Comment (erondpowell) [reply cloud/projects.md:91]: @AlexGaillard Good eye!

**Pull request merged**